Assets/Code/Scripts/LevelManagement/LevelManager.cs

```diff
@@ -57,6 +57,9 @@ public void LoadNextPuzzle()
             Debug.Log("LoadNextPuzzle");
             _currentPuzzleIndex++;
             transform.Find("Puzzle_" + _currentPuzzleIndex).gameObject.SetActive(true);
+            
+            EventManager.TriggerEvent("save_checkpoint_feedback", "Puzzle_" + _currentPuzzleIndex);  //Send to the feedback the checkpoint
```

Comment:
@TizioMaurizio, I wasn't able to distinguish the areas, do you know what to write in the message event in order to solve this problem?